shared/chat/conversation/attachment-get-titles/index.tsx

```diff
@@ -115,6 +115,7 @@ class GetTitles extends React.Component<Props, State> {
                 placeholder={titleHint}
                 multiline={true}
                 rowsMin={2}
+                padding="tiny"
```

Comment:
when we hoist up a style into a prop we need to blacklist it. see https://github.com/keybase/client/blob/master/shared/common-adapters/icon.d.ts#L8